frontend/simple/views/containers/Proposals.vue

```diff
@@ -0,0 +1,205 @@
+<template>
+  <dashboard-section title="Proposals">
+
+    <!--  Start Original Voting Banner to delete soon -->
+    <voting-banner v-for="proposal in Object.values(proposals)"
+      :proposal="proposal"
+    />
+    <!--  End Original Voting Banner to delete soon -->
+
+    <a class="gi-showAll"
+      v-if="groupProposals.alreadyVoted.length"
+      @click.prevent="showOtherProposals = true"
+    >
+      <i18n>Show all</i18n>
+    </a>
+
+    <!-- REVIEW: Not sure about these banners. -->
+
+    <!--
+    <h4 class="title is-size-5 notification is-warning gi-is-banner gi-notify"
+      v-if="groupProposals.notVoted.length"
+    >
+      <i18n>These are waiting for your vote!</i18n>
+    </h4>
+    -->
+
+    <i18n class="notification gi-is-banner gi-notify" v-if="allVoted">Cool, you already voted on all proposals.</i18n>
+
+    <voting
+      v-for="proposal in groupProposals.notVoted"
+      :proposal="proposal"
+      :onVoteAgainst="handleVoteAgainst"
+      :onVotedFor="handleVoteFor"
+    />
+
+    <voting
+      v-for="proposal in groupProposals.own"
+      :proposal="proposal"
+      :handleCloseProposal="onCloseProposal"
+    />
+
+    <div class="modal"
+      :class="{ 'is-active': showOtherProposals }"
+      v-if="groupProposals.alreadyVoted.length"
+    >
+      <div class="modal-background"></div>
+      <div class="modal-content gi-is-large">
+        <h4 class="title is-3"><i18n>All Proposals</i18n></h4>
+        <voting
+          v-if="showOtherProposals"
```

Comment:
All proposals that the user already voted on, are on a Modal as @taoeffect suggested, under a "Show All" link on the top right corner.

![image](https://user-images.githubusercontent.com/14869087/41033946-09b5997e-6980-11e8-9453-10778825f3bb.png)